src/game.cpp

```diff
@@ -5863,6 +5862,7 @@ void game::print_fields_info( const tripoint &lp, const catacurses::window &w_lo
             mvwprintz( w_look, point( column, ++line ), cur.color(), cur.name() );
         }
     }
+    mvwprintz( w_look, point( column, ++line ), c_white, "\n" );
```

Comment:
Printing a newline character will not have any visible effect. Just incrementing `line` will do it.